scripts/run-clang-format.sh

```diff
@@ -4,5 +4,5 @@ set -eu
 
 cd $(git rev-parse --show-toplevel)
 
-git ls-files | grep -e '\.cc$\|\.h$' | xargs -P4 clang-format -i
+git ls-files | grep -e '\.cc$\|\.h$|\.cpp$|\.hpp$' | xargs -P4 ${CLANG_FORMAT_BIN:-clang-format} -i
```

Comment:
Could you remove `grep` by `git ls-files -- '**/*.h' '**/*.cc' '**/*.hpp' '**/*.cpp'`?